frontend/src/components/common/table/table.tsx

```diff
@@ -1,21 +1,42 @@
+import { ReactComponent as DeleteIcon } from 'assets/icons/trash-can-solid.svg';
```

Comment:
We already have common icon component, declare new icon there and reuse Icon component

Reply:
agreed